src/global.f90

```diff
@@ -1859,6 +1858,8 @@ subroutine wrtend
   write(iunit,'(" vcasingtol  = ",es23.15       )') vcasingtol
   write(iunit,'(" vcasingits  = ",i9            )') vcasingits
   write(iunit,'(" vcasingper  = ",i9            )') vcasingper
+  write(iunit,'(" vcnt        = ",i9            )') vcnt
```

Comment:
I would like to suggest keeping the spelling/case the same as in the rest of the code, i.e., `vcNt` and `vcNz`